src/AccessibilityInsights.Extensions.AzureDevOps/FileIssue/FileIssueHelpers.cs

```diff
@@ -224,7 +225,7 @@ internal static string RemoveInternalHTML(string inputHTML, string keyText)
             {
                 foreach (IHTMLDOMNode divnode in divnodes)
                 {
-                    foreach (IHTMLDOMNode child in divnode.childNodes)
```

Comment:
devnode.childNodes no longer exposes GetEnumerator, so I had to rework it for enumeration